packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -57,6 +54,10 @@ export type PdfViewerProps = PdfDisplayProps & {
    * Callback for text layer info
    */
   setRenderedText?: (info: PdfRenderedText | null) => any;
+  /**
+   * URL of hosted PDF worker
+   */
+  pdfWorkerUrl?: string;
```

Comment:
If adding this url is a breaking change, then why is the prop optional? Will this component still work without the url? That goes for this component, but also any of the other components in this PR where we are passing through the url.